scripts/ansible/roles/linux/intel/vars/redhat/ootpa.yml

```diff
@@ -2,9 +2,9 @@
 # Licensed under the MIT License.
 
 ---
-intel_sgx_w_flc_driver_url: "https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/sgx_linux_x64_driver_1.33.bin"
```

Comment:
You can make these vars easier to handle by moving them to default variables and changing some parts to variables.

The advantage of having default vars over vars is the order of precedence of variables that Ansible uses, allowing some variables to overwrite others. E.g. Playbook vars will overwrite default vars, so one can quickly change the versions that are used by adding a playbook or runtime var without having to modify the role. To learn more, see https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

For example:
```
intel_driver_base_url: "https://download.01.org/intel-sgx/sgx-linux"
intel_driver_version: "2.10"
intel_driver_sgx_linux_version: "1.35"
intel_driver_sgx1_linux_version: "2.6.0_602374c"
intel_sgx_w_flc_driver_url: "{{ intel_driver_base_url }}/{{ intel_driver_version }}/distro/rhel8.1-server/sgx_linux_x64_driver_{{ intel_driver_sgx_linux_version }}.bin"
intel_sgx1_driver_url: "{{ intel_driver_base_url }}/{{ intel_driver_version }}/distro/rhel8.1-server/sgx_linux_x64_driver_{{ intel_driver_sgx1_linux_version }}.bin"
```
You could add more variables, but it depends on how you organized your roles and what tends to change or not.

Reply:
I like this solution, but given that we don't control the URLs, and the structure could change, I find that it's less error-prone to just copy and paste the url of the new drivers.